crates/egui_extras/src/loaders/image_loader.rs

```diff
@@ -19,16 +19,11 @@ impl ImageCrateLoader {
 }
 
 fn is_supported_uri(uri: &str) -> bool {
-    let Some(ext) = Path::new(uri).extension().and_then(|ext| ext.to_str()) else {
-        // `true` because if there's no extension, assume that we support it
-        return true;
-    };
-
     // Uses only the enabled image crate features
```

Comment:
In most cases it's fine, but I'm not sure if it's a good idea to remove this part.